robyn/__init__.py

```diff
@@ -82,6 +79,20 @@ def __init__(
         self.event_handlers: dict = {}
         self.exception_handler: Optional[Callable] = None
         self.authentication_handler: Optional[AuthenticationHandler] = None
+        self.included_routers: List[Router] = []
+
+    def init_openapi(self, openapi_file_path: Optional[str]) -> None:
+        if self.config.disable_openapi:
+            return
+
+        if self.openapi is None:
+            self.openapi = OpenAPI()
```

Comment:
will this ever happen?

Reply:
It shouldn't but it (probably temporarily pending the full routes refactor) keeps mypy happy because of the use of Optional in the argument list `openapi_file_path: Optional[str]`